hasura/metadata/databases/default/tables/public_locked_token_distribution_gifts.yaml

```diff
@@ -0,0 +1,51 @@
+table:
+  name: locked_token_distribution_gifts
+  schema: public
+object_relationships:
+  - name: locked_token_distribution
+    using:
+      foreign_key_constraint_on: locked_token_distribution_id
+  - name: profile
+    using:
+      foreign_key_constraint_on: profile_id
+insert_permissions:
+  - role: user
+    permission:
+      check:
+        locked_token_distribution:
+          epoch:
+            circle:
+              users:
+                _and:
+                  - profile:
+                      id:
+                        _eq: X-Hasura-User-Id
+                  - deleted_at:
+                      _is_null: true
+                  - role:
+                      _eq: 1
+      columns:
+        - earnings
+        - locked_token_distribution_id
+        - profile_id
+select_permissions:
+  - role: user
+    permission:
+      columns:
+        - id
+        - locked_token_distribution_id
+        - profile_id
+        - earnings
+      filter:
+        locked_token_distribution:
+          epoch:
+            circle:
+              users:
+                _and:
+                  - profile:
+                      id:
+                        _eq: X-Hasura-User-Id
+                  - deleted_at:
+                      _is_null: true
+                  - role:
+                      _eq: 1
```

Comment:
probably might not need the role restriction for querying

Reply:
good catch